fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfigHelper.java

```diff
@@ -191,4 +196,57 @@ private static JsonValue getPropertyFromTenantOrDefault(String propertyName) {
         
         return result;
     }
+    
+    /**
+     * Retrieves the search property restrictions.
+     *
+     * @param resourceType the resource type
+     * @param propertyType the property type
```

Comment:
probably worth documenting that this is expected be either "_include" or "_revinclude".  do any other values make sense?

Reply:
Added doc.